### PR TITLE
Added documentation for portal

### PR DESCRIPTION
**Pull request by goruha**
## What
* Document k8s `portal`

### CODE REVIEW

File: content/kubernetes-platform-services/dashboard/cluster-portal.md [hunk 1]
```diff
@@ -2,3 +2,144 @@
 title: "Cluster Portal"
 description: ""
 ---
+
+[Cluster portal](https://github.com/cloudposse/charts/tree/master/incubator/portal) allow to access
```
Comment (aknysh) [inline content/kubernetes-platform-services/dashboard/cluster-portal.md]: allows

File: content/kubernetes-platform-services/dashboard/cluster-portal.md [hunk 2]
```diff
@@ -2,3 +2,144 @@
...
+internal dashboards after authorization with third party [oAuth](https://en.wikipedia.org/wiki/OAuth) service.
```
Comment (aknysh) [inline content/kubernetes-platform-services/dashboard/cluster-portal.md]: after authorization with a third party OAuth service

File: content/kubernetes-platform-services/dashboard/cluster-portal.md [hunk 3]
```diff
@@ -2,3 +2,144 @@
...
+
+Cluster portal use [oauth2proxy](https://github.com/bitly/oauth2_proxy) as component
```
Comment (aknysh) [inline content/kubernetes-platform-services/dashboard/cluster-portal.md]: uses

File: content/kubernetes-platform-services/dashboard/cluster-portal.md [hunk 4]
```diff
@@ -2,3 +2,144 @@
...
+responsible for authentification.
+
+# Dependencies
+
+* [External DNS]({{< relref "kubernetes-backing-services/external-dns/external-dns.md" >}})
+* [Kube Lego]({{< relref "kubernetes-backing-services/tls-management/kube-lego-lets-encrypt.md" >}})
+
+# Install
+
+Starting installation the `portal` you need to define `hostname` used to access the portal.
```
Comment (aknysh) [inline content/kubernetes-platform-services/dashboard/cluster-portal.md]: To install the `portal`, you need to define `hostname`, which is a DNS name used to access the portal

File: content/kubernetes-platform-services/dashboard/cluster-portal.md [hunk 5]
```diff
@@ -3,10 +3,10 @@ title: "Cluster Portal"
 description: ""
 ---
 
-[Cluster portal](https://github.com/cloudposse/charts/tree/master/incubator/portal) allow to access
-internal dashboards after authorization with third party [oAuth](https://en.wikipedia.org/wiki/OAuth) service.
+[Cluster portal](https://github.com/cloudposse/charts/tree/master/incubator/portal) allows to access
+internal dashboards after authorization with a third party [oAuth](https://en.wikipedia.org/wiki/OAuth) service.
 
-Cluster portal use [oauth2proxy](https://github.com/bitly/oauth2_proxy) as component
+Cluster portal uses [oauth2proxy](https://github.com/bitly/oauth2_proxy) as component
```
Comment (osterman) [inline content/kubernetes-platform-services/dashboard/cluster-portal.md]:
```
Cluster portal use the Bitly [`oauth2-proxy`](https://github.com/bitly/oauth2_proxy) as an IAP. 
```

IAP will be highlighted by the glossary (identity aware proxy)

File: content/kubernetes-platform-services/dashboard/examples/values-portal.yaml [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+  rabbit:
+    name: "RabbitMQ"
+    endpoint: "http://popchest-rabbitmq.popchest-staging:15672"
+    external: false
+
+config:
+  title: "Staging Example"
+  brand: "Staging Cluster"
+  image:
+    favicon: https://d3r456tgct9p74.cloudfront.net/favicons/favicon.ico
```
Comment (osterman) [inline content/kubernetes-platform-services/dashboard/examples/values-portal.yaml]: Fix favicon

File: content/kubernetes-platform-services/dashboard/examples/values-portal.yaml [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+    url: https://d3r456tgct9p74.cloudfront.net/favicons/favicon-196x196.png
```
Comment (osterman) [inline content/kubernetes-platform-services/dashboard/examples/values-portal.yaml]: Fix favicon

File: content/kubernetes-platform-services/dashboard/examples/values-portal.yaml [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+    name: "Documentation"
+    endpoint: "https://docs.cloudposse.com/docs"
+    external: true
+  api:
+    name: "API Docs"
+    endpoint: "https://resources.staging.aws.popchest.io/api-docs/"
+    external: true
+  rabbit:
+    name: "RabbitMQ"
+    endpoint: "http://popchest-rabbitmq.popchest-staging:15672"
```
Comment (osterman) [inline content/kubernetes-platform-services/dashboard/examples/values-portal.yaml]: Drop `rabbit` from example.

File: content/kubernetes-platform-services/dashboard/examples/values-portal.yaml [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+    name: "Prometheus"
+    endpoint: "http://prom-prometheus.monitoring:9090"
+    external: false
+  docs:
+    name: "Documentation"
+    endpoint: "https://docs.cloudposse.com/docs"
+    external: true
+  api:
+    name: "API Docs"
+    endpoint: "https://resources.staging.aws.popchest.io/api-docs/"
```
Comment (osterman) [inline content/kubernetes-platform-services/dashboard/examples/values-portal.yaml]: Do not reference popchest

**Pull request merged**